src/database/services/severa-api-service.ts

```diff
@@ -93,33 +104,20 @@ export const CreateSeveraApiService = (): SeveraApiService => {
 
     /**
      * Gets work hours by userId
+     * 
+     * @param url url for fetching work hours
+     * @returns Work hours of a user with multiple queryparameters
      */
-    getWorkHoursBySeveraId: async (severaProjectId?:string, severaUserId?: string, severaPhaseId?: string, startDate?: string, endDate?: string) => {
-      let url: string;
-
-      if (severaProjectId) {
-        url = `${baseUrl}/v1/projects/${severaProjectId}/workhours`;
-      } else if (severaUserId) {
-        url = `${baseUrl}/v1/users/${severaUserId}/workhours`;
-      } else {
-        url = `${baseUrl}/v1/workhours`;
-      }
-      
-      const queryParams = new URLSearchParams();
-      if (startDate) queryParams.append("startDate", startDate);
-      if (endDate) queryParams.append("endDate", endDate);
-      if (queryParams.toString()) {
-        url += `?${queryParams.toString()}`;
-      }
-
-      const response = await fetch(url, {
+    getWorkHoursBySeveraId: async (url: string) => {
+      const response = await fetch(`${baseUrl}/v1/${url}`, {
```

Comment:
Maybe we should rename url, to something like endpointPath or whatever you think would be better